hardhat/contracts/WorldMap.sol

```diff
@@ -0,0 +1,68 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity >=0.8.0 <0.9.0;
+
+import "./PlanetFactory.sol";
+
+contract WorldMapCreator is PlanetFactory {
+
+    struct WorldMap {
+        uint256 worldIndex; // The ID for the world that was created
+        uint256 Length; // The vertical length of the world map
+        uint256 Breadth; // The breadth of the world map
+    }
+
+    mapping(uint256 => WorldMap) public existingWorlds;
+    mapping(uint => uint[]) public planetsInWorld;
+
+    uint planetIndex;
+
+    constructor() {
+        planetIndex = 0;
+    }
+
+    // function initialiseWorld () {
+        // TODO: consider using this function to call both
+        // _defineWorldMap and _addPlanets functions
+    // }
+
+    function defineWorldMap(uint256 _worldIndex, uint256 _length, uint256 _breadth) public {
```

Comment:
interesting idea that we can have multiple worlds

Reply:
could potentially be used for playing around with diff worlds from front-end? perhaps a 'test' one